packages/main/src/ColorPalette.ts

```diff
@@ -259,34 +275,80 @@ class ColorPalette extends UI5Element {
 		});
 	}
 
-	_onclick(e: MouseEvent) {
-		const target = e.target as ColorPaletteItem;
-		if (target.hasAttribute("ui5-color-palette-item")) {
-			this.selectColor(target);
+	get effectiveColorItems() {
+		let colorItems = this.colors;
+
+		if (this.popupMode) {
+			colorItems = this.getSlottedNodes<ColorPaletteItem>("colors");
```

Comment:
Do we need this since we don't use static area anymore?

Reply:
I think yes, as I am not sure how would I retrieve the slotted colors otherwise when in ColorPalettePopover, do you have any suggestions ?

Reply:
We have to use getSlottedNotes when we have transitive slots like the following case. The is discussion in topic team to enable getSlottedNotes for each slot by default but it's still ongoing.